aegis-web/yo/app/navProject.html

```diff
@@ -93,8 +93,15 @@
           Members
         </a>
       </li>
+      
+      <li ng-click="projectCtrl.goToProject()" ng-show="projectCtrl.cloak">
+        <a>
+          <span class="sub_icon fa fa-list fa-lg" style="width: 30px; display: inline-block; margin-right: 10px"></span>
```

Comment:
You didn't like the image i created?

Reply:
yes I like it. I kept awesome fonts for uniformity. I can insert yours

Reply:
Not every menu option use fonts for icon. 
It's up to you, if you like the fa-list better that the png i send you, keep it. Otherwise use the image. 
I haven't seen either of them in place so i can't judge.